MiniSim/Service/DeviceService.swift

```diff
@@ -235,7 +279,45 @@ extension DeviceService {
 
   static func getIOSDevices() throws -> [Device] {
     Thread.assertBackgroundThread()
+    let simulators = try getIOSSimulators()
+    let devices = try getIOSPhysicalDevices()
```

Comment:
This can fail if user doesn't have Xcode 15. 

Even if the physical devices fail we want to display simulators

Reply:
Would a guard statement with optional try and returning an empty array be good here?:
https://github.com/okwasniewski/MiniSim/blob/fa3503d984af87546eb980adca7dba38e0ea08db/MiniSim/Service/DeviceService.swift#L248-L251

Reply:
Yeah I think this should be good

Reply:
done